modules/Users/metadata/detailviewdefs.php

```diff
@@ -49,7 +49,9 @@
                      * To customise in an upgrade safe way, You need to create custom view instead.
                      * Then override UsersViewDetail::preDisplay().
                      */
-                    'buttons' => array (),
+                    'buttons' => array (
+                        array('customCode' => '<input title="'.translate('LBL_GENERATE_PASSWORD_BUTTON_TITLE','Users').'" class="button" LANGUAGE=javascript onclick="generatepwd(\'{$fields.id.value}\');" type="button" name="password" value="'.translate('LBL_GENERATE_PASSWORD_BUTTON_LABEL','Users').'">"')
```

Comment:
Can not use this here as stated by the above comments:

/**
                     * Actions for users are configured in modules/Users/views/view.detail.php
                     * This is to control security access to the actions based on the user and system preferences.
                     * To customise in an upgrade safe way, You need to create custom view instead.
                     * Then override UsersViewDetail::preDisplay().
                     */